data/core/terrain-graphics/new-macros.cfg

```diff
@@ -2383,7 +2390,7 @@ X*#endarg
             base=90,144
             center=90,144
             [variant]
-                tod=first_watch,midnight,second_watch,dusk,dusk2,short_dark,long_dark1,long_dark2,long_dark3,long_dark4
+                tod={TIME}
```

Comment:
I guess all the removed times here are based on the idea of not leaving the lights on all night?

Reply:
I guess.  I don't know why I chose the old ones, why was 'midnight' in there?

Reply:
You're asking me? :P